src/main/webapp/app/core/navbar/navbar.component.ts

```
@@ -604,6 +608,10 @@ export class NavbarComponent implements OnInit, OnDestroy {
                 const updatedLink = currentPath.replace('/submissions/', '/scores/');
                 this.addTranslationAsCrumb(updatedLink, 'submissions');
                 break;
+            case 'tutorial-lectures':
+                const tutorialGroupsLink = currentPath.replace('/tutorial-lectures/', '');
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)